ca_on_toronto/events-incremental.py

```diff
@@ -155,7 +155,7 @@ def is_council(event):
                     for i, item in enumerate(agenda_items):
 
                         a = e.add_agenda_item(item['title'])
-                        a.add_classification(item['type'].lower())
+                        #a.add_classification(item['type'].lower())
```

Comment:
I'm waiting on merge of https://github.com/opencivicdata/pupa/pull/235, and then can fix things so we won't need to comment this out... Hopefully we can get that sorted before the Tues hack night

cc: @jamesturk